docs/tutorials/projects/verification-system.md

```diff
@@ -0,0 +1,240 @@
+---
+title: "Create a Facial Verification System"
+linkTitle: "Verification System"
+weight: 40
+type: "docs"
+description: "Configure a verification system module to implement a facial verification security system on your smart machine."
+image: "/tutorials/verification-system/disarmed.png"
+imageAlt: "Bijan disarming the facial verification system."
+images: ["/tutorials/verification-system/disarmed.png"]
+tags: ["mlmodel", "vision", "services", "security", "canera"]
+authors: []
+languages: []
+viamresources: ["mlmodel", "vision", "camera"]
+level: "Intermediate"
+date: "28 November 2023"
+cost: "0"
+no_list: true
+# SMEs: Bijan Haney
+---
+
+In the [registry](https://app.viam.com/module/viam-labs/verification-system), Viam Labs provides a model of [vision service classifier](/ml/vision/classification/) called `verification-system`.
+Follow this tutorial to learn how to configure this module and set up your own security system to detect, verify, and alarm based on specified detections from nested [vision service detectors](/ml/vision/detection/).
+
+You can use whatever combination of detectors and camera you would like with this module, but follow along with our recommendations to layer an [`mlmodel` person detector](#configure-an-mlmodel-person-detector) and [facial detector](#configure-a-facial-detector) together, creating a security system with facial recognition.
+
+{{< alert title="Requirements" color="note" >}}
+Before configuring your camera, you must [create a robot](/fleet/machines/#add-a-new-robot).
+{{< /alert >}}
+
+## Configure a camera
+
+Navigate to the **Config** tab of your machine's page on the [Viam app](https://app.viam.com).
+Configure the camera you want to provide an image stream for your verification system on.
+
+We configured ours as a webcam named `"my-webcam"`.
+Follow [these instructions](/build/configure/components/camera/webcam/) to do so.
+
+## Configure an `mlmodel` person detector
+
+Create a detector to detect when people are in your verification system camera's field of vision:
+
+- [Capture images from your camera](/data/capture/#configure-data-capture-for-individual-components) with Viam's data management service, then [create a dataset](/data/dataset/#create-a-dataset-and-add-data).
+- Label your data with [bounding boxes](/data/dataset/#bounding-boxes) labeled as `Person` where a person is present to create an object detection model for people.
+- [Train a model on your dataset](/ml/train-model/).
+- [Deploy the model](/ml/deploy/) to your machine.
```

Comment:
A little more detail on each step would be helpful . You can explain that you need to take images of the people who should be able to disarm the system or just people (?) and so on

Reply:
Ok!